infinite_games/events/base.py

```diff
@@ -756,7 +756,7 @@ def mark_event_as_exported(self, pe: ProviderEvent) -> bool:
         conn.close()
         return False
 
-    def mark_submissions_as_exported(self) -> bool:
+    def mark_submissions_as_exported(self, interval_start_minutes: int, previous_interval_start_minutes) -> bool:
```

Comment:
I would rather use backoff than custom code for retries

Reply:
with backoff we had an hidden logs issue, was hard to debug as mentioned, so here i prefer for explicit logic. and this one is not only for retries. in case if validator is not active or something we have to cover previous intervals

Reply:
is this explicit logic unit tested? because otherwise I think it is preferably to use code from standard libraries